lib/compiler/build_runner.zig

```diff
@@ -1323,6 +1330,7 @@ fn usage(b: *std.Build, out_stream: anytype) !void {
         \\  --search-prefix [path]       Add a path to look for binaries, libraries, headers
         \\  --sysroot [path]             Set the system root directory (usually /)
         \\  --libc [file]                Provide a file which specifies libc paths
+        \\  --search-prefix-exact [file] Provide a file which specifies exact search paths
```

Comment:
I think I would prefer `--search-paths` as the name for this flag. After all, you're supplying a file that specifies paths relative to CWD, though usually they'll just be absolute paths. It doesn't seem like there's really any prefixing going on when using this.

(I think we ought to also consider renaming `--libc` to `--libc-paths` and make it use a ZON file, but that's of course a separate change.)

Reply:
done